modules/table/tech-notes/rebalance.md

```diff
@@ -1,92 +1,80 @@
-- Metastore event about change in `<global>.baseline`
-- Metastore event about changes in `<tableScope>.replicas`
+- Metastore event about changes in `<tableScope>.replicas` (See `org.apache.ignite.internal.table.distributed.TableManager.onUpdateReplicas`)
 
 **Pseudocode**:
-```
-onBaselineEvent:
-    for table in tableCfg.tables():
-        for partition in table.partitions:
-            <inline metastoreInvoke>
-            
+```         
 onReplicaNumberChange:
     with table as event.table:
         for partitoin in table.partitions:
             <inline metastoreInvoke>
 
 metastoreInvoke: // atomic metastore call through multi-invoke api
-    if empty(partition.assignments.change.trigger.revision) || partition.assignments.change.trigger.revision < event.revision:
+    if empty(partition.change.trigger.revision) || partition.change.trigger.revision < event.revision:
         if empty(partition.assignments.pending) && partition.assignments.stable != calcPartAssighments():
             partition.assignments.pending = calcPartAssignments() 
-            partition.assignments.change.trigger.revision = event.revision
+            partition.change.trigger.revision = event.revision
         else:
             if partition.assignments.pending != calcPartAssignments
                 partition.assignments.planned = calcPartAssignments()
-                partition.assignments.change.trigger.revision = event.revision
+                partition.change.trigger.revision = event.revision
             else
                 remove(partition.assignments.planned)
     else:
         skip
 ```
 
 ## Start new raft nodes and initiate change peers (2)
-**Trigger**: Metastore event about new `partition.assignments.pending` received
+**Trigger**: Metastore event about new `partition.assignments.pending` received (See corresponding listener for pending key in `org.apache.ignite.internal.table.distributed.TableManager.registerRebalanceListeners`)
 
 **Steps**:
 - Start all new needed nodes `partition.assignments.pending / partition.assignments.stable` 
-- After successful starts - check if current node is the leader of raft group (leader response must be updated by current term) and `changePeers(leaderTerm, peers)`. `changePeers` from old terms must be skipped.
+- After successful starts - check if current node is the leader of raft group (leader response must be updated by current term) and run `RaftGroupService#changePeersAsync(leaderTerm, peers)`. `RaftGroupService#changePeersAsync` from old terms must be skipped.
 
 **Result**:
 - New needed raft nodes started
 - Change peers state initiated for every raft group
 
-## When changePeers done inside the raft group - stop all redundant nodes
-**Trigger**: When leader applied new Configuration with list of resulting peers `<applied peer>`, it calls `onChangePeersCommitted(<applied peers>)`
+## When RaftGroupService#changePeersAsync done inside the raft group - update stable key and stop all redundant nodes
```

Comment:
Could you add some words about assignments' configuration update and further updates of raft clients for table?